db/migrations/0003_alter_movie_actors_alter_movie_genres.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2024-11-01 18:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(null=True, to='db.Actor'),

```

Comment:
The `null=True` parameter is not valid for ManyToManyField. If you want to allow a movie to have no actors, you should use `blank=True` instead.